src/main/java/uk/gov/hmcts/ccd/domain/service/search/PaginatedSearchMetaDataOperation.java

```diff
@@ -21,6 +20,10 @@ public PaginatedSearchMetaDataOperation(@Qualifier(CachedCaseDetailsRepository.Q
     }
 
     public PaginatedSearchMetadata execute(MetaData metaData, Map<String, String> criteria) {
+
+        if (!metaData.validateAndsConvertReference()) {
+            return new PaginatedSearchMetadata();
```

Comment:
```suggestion
            return PaginatedSearchMetadata.EMPTY;
```
where EMPTY is a constant in PaginatedSearchMetadata:
`public static final EMPTY = new PaginatedSearchMetadata();`

Reply:
DONE